app/templates/index.html

```diff
@@ -138,8 +141,12 @@
                             if (sformat == "json") {
                                 $(".code").attr("class", "language-json");
                                 response = JSON.stringify(response, null, 4);
-                            } else
+                            } else if (sformat == "xml"){
                                 response = new XMLSerializer().serializeToString(response);
+                            }else{
+                                console.log("CSV Respose here");
+                                return response;
```

Comment:
Please look into it. Is really the csv response will be returned like as normal ?

Reply:
Nope, This PR is not complete